source/tyr/tyr/helper.py

```diff
@@ -191,3 +192,24 @@ def get_named_arg(arg_name, func, args, kwargs):
             return args[idx]
         else:
             return None
+
+
+def get_tempfile(file_storage):
+    """
+    Save stream file in temp directory
+    :param file_storage: stream file
+    :return: fileneme
+    """
+    tmp_file = os.path.join(tempfile.gettempdir(), file_storage.filename)
+    file_storage.save(tmp_file)
+    return tmp_file
+
+
+def rm_tempfile(filename):
```

Comment:
no it should be handled by  the different task no ? (isn't there one moving the file to the backup ?)